src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtRegsDsa.java

```diff
@@ -11,9 +11,9 @@
 public class ExtRegsDsa {
 
     /**
-     * Defines the contract for bidrequest.regs.ext.dsa.required
+     * Defines the contract for bidrequest.regs.ext.dsa.dsarequired
      */
-    Integer required;
+    Integer dsarequired;
```

Comment:
Please follow Java code style and use @JsonProperty or use Jackson naming strategy